config/locales/en/views.yml

```diff
@@ -21,6 +21,12 @@ en:
     snippet:
         characteristic_needs_feature: 'Knowledge Map characteristics can only be associated from a subject.'
         feature:
+            browse_related:
+              one: 'You can browse this subordinate subject as well as its superordinate subjects with the tree below.'
+              other: 'You can browse those subordinate subjects as well as its superordinate subjects with the tree below.'
+            browse_summary:
```

Comment:
Better to have only: 1) words, 2) self-contained phrases or 3) sentences, but not half a sentence here.